Source/Core/Core/IOS/Network/SSL.cpp

```diff
@@ -21,6 +21,12 @@
 #include "Core/HW/Memmap.h"
 #include "Core/IOS/Network/Socket.h"
 
+namespace
+{
+// Dirty workaround to remove SNI from Client Hello
+int mbedtls_net_send_without_sni(void* ctx, const unsigned char* buf, size_t len);
```

Comment:
Any reason that this anonymous namespace cant be within the device namespace? (and also have the implementation there as well)

Maybe also expand the comment to explain why the workaround needs to exist.